arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1597 @@
+      "properties": {
+        "suffix": {
+          "type": "string",
+          "description": "the optional suffix for the Data Lake Store account."
+        }
+      },
+      "description": "Data Lake Store account properties information."
+    },
+    "DataLakeStoreAccountInfo": {
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the account name of the Data Lake Store account to add to the Data Lake Analytics account being created."
+        },
+        "properties": {
+          "$ref": "#/definitions/DataLakeStoreAccountInfoProperties",
+          "description": "the properties associated with this Data Lake Store account.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "description": "Data Lake Store account information."
+    },
+    "DataLakeAnalyticsAccountListStorageAccountsResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageAccountInfo"
+          },
+          "readOnly": true,
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "Azure Storage Account list information."
+    },
+    "DataLakeAnalyticsAccountListDataLakeStoreResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataLakeStoreAccountInfo"
+          },
+          "readOnly": true,
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "Data Lake Account list information."
+    },
+    "DataLakeAnalyticsAccountProperties": {
+      "required": [
+        "defaultDataLakeStoreAccount",
+        "dataLakeStoreAccounts"
+      ],
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the provisioning status of the Data Lake Analytics account.",
+          "enum": [
+            "Failed",
+            "Creating",
+            "Running",
+            "Succeeded",
+            "Patching",
+            "Suspending",
+            "Resuming",
+            "Deleting",
+            "Deleted"
+          ],
+          "x-ms-enum": {
+            "name": "DataLakeAnalyticsAccountStatus",
+            "modelAsString": false
+          }
+        },
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the state of the Data Lake Analytics account.",
+          "enum": [
+            "active",
+            "suspended"
```

Comment:
Good catch, it should be in PascalCase (first letter upper case). Fixing this across the board.